cmd/tracee-ebpf/main.go

```diff
@@ -148,7 +148,8 @@ func main() {
 			}
 
 			printerConfig.ContainerMode = containerMode
-			cfg.Output = &output
+			cfg.Output = &output.OutputConfig
+			logger.SetWriter(output.LogFile)
```

Comment:
Programmatically we can set it through SetBase. 
From my experience working  on this package now I think we should consider moving away from all the implicit env vars, since it made logic much more complex with a single global Init() point of entry.

Reply:
> Programmatically we can set it through SetBase. From my experience working on this package now I think we should consider moving away from all the implicit env vars, since it made logic much more complex with a single global Init() point of entry.

Should env vars be checked before the logger.Init() (outside the logger)?

Reply:
I think we should drop the env vars, it wasn't a good design decision in hindsight, library consumers should Init the logger on their own if they want non default behavior (this is what we do in other aqua products too).